bundles/binding/org.openhab.binding.zwave/database/benext/doorsensor.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+	<Model>doorSensor</Model>
+	<Label lang="en">BeNext Door Sensor</Label>
+	<CommandClasses>
+		<Class>
+			<id>0x20</id>
+		</Class>		
+		<Class>
+			<id>0x30</id>
+		</Class>
+		<Class>
+			<id>0x31</id>
+		</Class>
+		<Class>
+			<id>0x70</id>
+		</Class>
+		<Class>
+			<id>0x71</id>
+		</Class>
+		<Class>
+			<id>0x72</id>
+		</Class>
+		<Class>
+			<id>0x73</id>
+		</Class>
+		<Class>
+			<id>0x84</id>
+		</Class>
+		<Class>
+			<id>0x85</id>
+		</Class>
+		<Class>
+			<id>0x86</id>
+		</Class>
+	</CommandClasses>
+	<Configuration>
+		<Parameter>
+			<Index>1</Index>
+			<Type>byte</Type>
+			<Minimum>0</Minimum>
+			<Maximum>256</Maximum>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Label lang="en">Reset to factory settings</Label>
+			<Help lang="en">Set all configuration values to default values (factory settings).</Help>
+		</Parameter>
+		<Parameter>
+			<Index>2</Index>
+			<Type>list</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Label lang="en">Configure what the external contact sends when triggered.</Label>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">Send a alarm report with type 2</Label>				
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Send a Basic set frame to all nodes in association group 2.</Label>				
+			</Item>			
+		</Parameter>
+		<Parameter>
+			<Index>5</Index>
+			<Type>list</Type>
+			<Default>1</Default>
+			<Size>1</Size>
+			<Label lang="en">The operating mode.</Label>			
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Normal operating mode.</Label>
+			</Item>
+			<Item>
+				<Value>2</Value>
+				<Label lang="en">If any mode other then 3, that value will be reported after a get but will be handled in SW as mode 1.</Label>
+			</Item>
+			<Item>
+				<Value>3</Value>
+				<Label lang="en">Z-Wave chip is always on to request e.g. version or manufacturer id.</Label>
+			</Item>
+			<Item>
+				<Value>4</Value>
+				<Label lang="en">If any mode other then 3, that value will be reported after a get but will be handled in SW as mode 1.</Label>
```

Comment:
Why are 2 and 4 the same? If they really do do the same thing, then why not just have one of them in the list?

Ideally, it would be best to keep the option labels short so that they fit in the options box, and then add any additional text into the `<Help>` tag.  This comment is also true for the other long labels.